bedrock/privacy/templates/privacy/archive/firefox-2006-10.html

```diff
@@ -11,7 +11,7 @@
 {% block article %}
   <article class="section-content" itemscope itemtype="http://schema.org/Article">
     <header>
-      <img class="logo" src="/media/img/privacy/logo-firefox.png" alt="">
+      <img class="logo" src="/media/img/logos/firefox/quantum/logo-xl.png" alt="">
```

Comment:
This logo appears blurry to me here, maybe the hi res image helper would help but I wouldn't call it a blocker.

Reply:
Yeah but I just realized these are out dated policies and should have the old-old logo on them. And I do think that is a blocker.